pages/tool_assembly/csc_assembly.md

```diff
@@ -46,24 +46,24 @@ You can access all CSC services through several secure authentication methods. S
 
-Supercomputers Puhti and Mahti can be used for larger scale analysis and simulations. They will soon be accompanied with the world-class supercomputer {% tool "lumi" %}. Pouta and Rahti cloud computing services offer more flexibility, allowing the user to manage the infrastructure. CSC's computers have a wide range of [preinstalled scientific software and databases](https://research.csc.fi/bioscience-programs) with usage instructions. 
+CSC is hosting  EuroHPC's world-class supercomputer [LUMI](https://www.lumi-supercomputer.eu/), which is available for researchers across Europe for projects requiring extreme computing capacity. LUMI is one of the world's most competitive supercomputers and one of the most advanced platforms for artificial intelligence. In addition, you can utilize two national supercomputers Puhti and Mahti for medium to large scale simulations. Pouta and Rahti cloud computing services offer more flexibility, allowing the user to manage the infrastructure. CSC’s computers have a wide range of [preinstalled scientific software and databases](https://research.csc.fi/bioscience-programs) with usage instructions.
 
-This summer, CSC will be releasing beta versions of new services for sensitive data management: Sensitive Data Desktop (SD Desktop) and Sensitive Data Connect (SD Connect). Sensitive Data Submit (SD Submit) will be available later this year. The new Sensitive Data Services are designed to facilitate collaborative research across Finland and between Finnish academics and their collaborators. [SD Desktop](https://research.csc.fi/-/sd-desktop) is a service that allows a user and their authorized colleagues to access a private computing environment workspace via a web browser and analyze the data within a secure cloud. [SD Connect](https://research.csc.fi/-/sd-connect) allows you to collect, organize and share your encrypted sensitive data in a secure manner via web browser.
+For sensitive data management, there is available [SD Connect](https://research.csc.fi/-/sd-connect) and [SD Desktop](https://research.csc.fi/-/sd-desktop) services. The Sensitive Data Services are designed to facilitate collaborative research across Finland and between Finnish academics and their collaborators.  SD Connect allows you to collect, organize and share your encrypted sensitive data in a secure manner via web browser or programmatically.  SD Desktop is a service that allows a user and their authorized colleagues to access a private computing environment workspace via a web browser and analyze the data within a secure cloud. A restricted version of SD Desktop is also available for processing health and social data for secondary use in compliance with the Finnish law and Findata regulation.
```

Comment:
```suggestion
For management of sensitive data, the [SD Connect](https://research.csc.fi/-/sd-connect) and [SD Desktop](https://research.csc.fi/-/sd-desktop) services are available. The Sensitive Data Services are designed to facilitate collaborative research across Finland and between Finnish academics and their collaborators.  SD Connect allows you to collect, organise and share your encrypted sensitive data in a secure manner via web browser or programmatically.  SD Desktop is a service that allows a user and their authorized colleagues to access a private computing environment workspace via a web browser and analyse the data within a secure cloud. A restricted version of SD Desktop is also available for processing health and social data for secondary use in compliance with the Finnish law and Findata regulation.
```

Reply:
>in compliance with the Finnish law and Findata regulation.

Consider adding links for these.

Reply:
We have more links on the national page (Finland). But thank you for checking the text (we did try to check with 6 eyes already :D)

Reply:
> We have more links on the national page (Finland). 

You could just say that more info is provided on the national page and link it directly. Up to you.

>But thank you for checking the text

No problem, it was a quick check. The more challenging bit is to move all the existing content to the new syntax for tools. I am trying to do it little by little as I get pages to review.